### PR TITLE
style: Add prettier plugin that formats JSDoc comments and run it.

### PR DESCRIPTION
**Pull request by melink14**
Update eslint config to prefer `@returns` over `@return` to match formatter.
This required mirroring some unrelated `valid-jsdoc` config due to configs not being merged.
Use two newlines to force linebreak.

---
This change is [Reviewable](https://reviewable.io/reviews/melink14/rikaikun/217)

### CODE REVIEW

File: extension/background.js [hunk 1]
```diff
@@ -118,9 +115,9 @@ function initializeConfigFromCloudOrLocalStorageOrDefaults(cloudStorage) {
   initConfig('ttsEnabled', false);
 
   /**
-   * Set kanjiInfo option values
-   * Check each key in case there are new types of info to be added to the
-   * config. TODO: Consider a solution that doesn't require this loop.
+   * Set kanjiInfo option values Check each key in case there are new types of info
```
Comment (Stephie) [inline extension/background.js:118]: The line break disappeared!

File: extension/background.js [hunk 2]
```diff
@@ -11,10 +11,7 @@ chrome.runtime.onMessage.addListener(function (request, sender, response) {
       const e = rcxMain.search(request.text, request.dictOption);
       response(e);
       break;
-    /**  case 'nextDict':
-			 console.log('nextDict');
-			 rcxMain.nextDict();
-			 break;*/
+    /** Case 'nextDict': console.log('nextDict'); rcxMain.nextDict(); break; */
```
Comment (Stephie) [inline extension/background.js:14]: Is there a way to preserve the formatting of commented-out code?

**Review by Stephie — CHANGES_REQUESTED** [review]

**Review by melink14 — COMMENTED** [review]
Fixed.

> *[Reviewable](https://reviewable.io/reviews/melink14/rikaikun/217#-:-MFsXafR-QOf7BwgcQRb:bt2vr1b)* status: 0 of 8 files reviewed, all discussions resolved

**Review by Stephie — APPROVED** [review]
> Reviewed 8 of 13 files at r2.
> *[Reviewable](https://reviewable.io/reviews/melink14/rikaikun/217)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**